hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/request/security/OMCancelDelegationTokenRequest.java

```diff
@@ -70,10 +91,15 @@ public OMClientResponse validateAndUpdateCache(OzoneManager ozoneManager,
 
     OMMetadataManager omMetadataManager = ozoneManager.getMetadataManager();
 
+    AuditLogger auditLogger = ozoneManager.getAuditLogger();
+    Map<String, String> auditMap = buildTokenAuditMap(getToken());
```

Comment:
Nit: we can assign `getToken()` to a local variable and use it when assigning `ozoneTokenIdentifier` below.